flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -1429,5 +1457,106 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
     }
 
     /* END OF MCDU AOC MESSAGE SYSTEM */
+    /* WEBSOCKET */
+
+    /**
+     * Attempts to connect to a websocket server on 127.0.0.1:8080
+     */
+    connectWebsocket() {
+        if (this.socket) {
+            this.socket.close();
+            this.socket = undefined;
+        }
+
+        if (this.socketTimeout) {
+            clearTimeout(this.socketTimeout);
+        }
+
+        const url = "127.0.0.1:8080";
```

Comment:
Do you want me to add a commit with a flyPad setting for this? Maybe also a toggle to enable/disable?

Reply:
I already made one previously (see earlier commits) but ended up taking it out because I wanted to make it work without having to configure anything in the sim (just run server.exe and open the url). The only issue with this is the user might want to change the port in the event that it is taken, so I am considering adding it back.

Reply:
Maybe we could make it default to `http://localhost:8080` and enabled... and power users can choose a different URL in the flypad if there is a port clash?

Reply:
Alright, I added an EFB option for the web socket url and a command line argument for the the server port.